specification/azsadmin/resource-manager/fabric/Microsoft.Fabric.Admin/preview/2016-05-01/FabricLocation.json

```diff
@@ -93,40 +93,6 @@
                     "nextLinkName": "nextLink"
                 },
                 "x-ms-odata": "#/definitions/FabricLocation"
-            },
-            "put": {
-                "x-ms-examples": {
-                    "Creates or updates a fabric location.  This will fail if called outside deployment.": {
-                        "$ref": "./examples/FabricLocation/Put.json"
-                    }
-                },
-                "tags": [
-                    "FabricLocations"
-                ],
-                "description": "Creates or updates a fabric location.  This will fail if called outside deployment.",
-                "operationId": "FabricLocations_Create",
-                "parameters": [
-                    {
-                        "$ref": "Fabric.json#/parameters/SubscriptionIdParameter"
-                    },
-                    {
-                        "$ref": "Fabric.json#/parameters/ResourceGroupParameter"
-                    },
-                    {
-                        "$ref": "Fabric.json#/parameters/ApiVersionParameter"
-                    },
-                    {
-                        "$ref": "#/parameters/FabricLocationObjectParameter"
```

Comment:
Remove the local parameter not longer been used.
#/parameters/FabricLocationObjectParameter

Reply:
done